great_expectations/expectations/core/expect_column_values_to_be_in_set.py

```diff
@@ -193,93 +196,87 @@ def _atomic_prescriptive_template(
-        params = substitute_none_for_missing(
-            configuration.kwargs,
-            [
-                "column",
-                "value_set",
-                "mostly",
-                "parse_strings_as_datetimes",
-                "row_condition",
-                "condition_parser",
-            ],
+        renderer_configuration.add_param(
+            name="column",
+            schema_type="string",
+        )
+        renderer_configuration.add_param(
+            name="value_set",
+            schema_type="array",
+        )
+        renderer_configuration.add_param(
+            name="mostly",
+            schema_type="number",
+        )
+        renderer_configuration.add_param(
+            name="mostly_pct",
+            schema_type="string",
+        )
+        renderer_configuration.add_param(
+            name="parse_strings_as_datetimes",
+            schema_type="boolean",
+        )
+        renderer_configuration.add_param(
+            name="row_condition",
+            schema_type="string",
+        )
+        renderer_configuration.add_param(
+            name="condition_parser",
+            schema_type="string",
         )
-        params_with_json_schema = {
-            "column": {"schema": {"type": "string"}, "value": params.get("column")},
-            "value_set": {
-                "schema": {"type": "array"},
-                "value": params.get("value_set"),
-            },
-            "mostly": {"schema": {"type": "number"}, "value": params.get("mostly")},
-            "mostly_pct": {
-                "schema": {"type": "string"},
-                "value": params.get("mostly_pct"),
-            },
-            "parse_strings_as_datetimes": {
-                "schema": {"type": "boolean"},
-                "value": params.get("parse_strings_as_datetimes"),
-            },
-            "row_condition": {
-                "schema": {"type": "string"},
-                "value": params.get("row_condition"),
-            },
-            "condition_parser": {
-                "schema": {"type": "string"},
-                "value": params.get("condition_parser"),
-            },
-        }
 
-        if params["value_set"] is None or len(params["value_set"]) == 0:
+        params: RendererParams
+        params = renderer_configuration.params
+
+        if not params.value_set or len(params.value_set.value) == 0:
             values_string = "[ ]"
         else:
-            for i, v in enumerate(params["value_set"]):
-                params[f"v__{str(i)}"] = v
+            for i, v in enumerate(params.value_set.value):
+                renderer_configuration.add_param(
+                    name=f"v__{str(i)}", schema_type="string", value=v
+                )
+
+            params = renderer_configuration.params
 
             values_string = " ".join(
-                [f"$v__{str(i)}" for i, v in enumerate(params["value_set"])]
+                [f"$v__{str(i)}" for i, v in enumerate(params.value_set.value)]
```

Comment:
Whenever we want to do something with the `.value` other than evaluate it's truthiness, we need to use the `.value`. This is because the `.value` could be anything (it's often an Expectation kwarg), and it's not advisable to overload every dunder method imaginable.